process/sandbox.md

```diff
@@ -0,0 +1,94 @@
+
+Therefore the advantages of being in the Sandbox apply prior to application for Incubation, ie.:
+
+1. That a project has a legally neutral home that is stable and known
+
+2. And that a project may attain Incubation-level success faster:
+
+    1. Due to public visibility and association with the OpenSSF mission
+    2. Through alignment with other OpenSSF projects (if and only if desired)
+    3. Via OpenSSF Service Desk etc.
+
+3. The OpenSSF will help projects adopt good principles of governance
+
+### Neutral Home
+
+A neutral home for your project increases the willingness of developers from other companies and independent developers to collaborate, contribute, and become committers. Neutrality requires that projects contribute their trademark to OpenSSF so that:
+
+* No company is favored over any other
+* OpenSSF ensures project governance is transparent and fair for everyone.
+
+### Clarifying Marketing Expectations
+
+All open source projects in some sense enjoy a level of promotion from community, user enthusiasm, sponsoring organisations and so on. Please note that in this section we discuss marketing as a measurable financial investment into OpenSSF projects from the OpenSSF marketing budget and staff.
+
+To date the OpenSSF has invested in marketing to educate users and grow awareness of cloud native purpose and benefits, to foster community, and to accelerate production use of projects.  These investments fall into at least three types:
+
+* Developer community support: hangouts, meetups, events and (some) conferences
+* Digital marketing: help with online content, interactive tutorials, webinars, and social
+* Product marketing: conference promotion, landscape, certification, case studies, AR/PR
+
+Since the Sandbox is for early stage, sandbox projects will receive minimal marketing support from the foundation.  The Sandbox group as a whole may be promoted from time to time.
+
+The OpenSSF will lean towards developer community support and the OpenSSF service desk, to help discovery and initial steps towards OpenSSF Incubation.  There will only be limited OpenSSF investment in Digital and Product marketing for individual Sandbox projects, and OpenSSF-funded content should be factual and informative.
+
+Some key points:
+
+* Sandbox projects will be listed separately from other OpenSSF projects (cncf.io/sandbox)
+* They will not be prominently listed at our events or issued a press release
+* Reviewed on an annual basis; submit a report to the TAC for review
+* OpenSSF Sandbox projects can stay in the sandbox indefinitely
+
+### Application into Sandbox
+
+To apply for inclusion into the Sandbox, projects should use [this form](https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit).
```

Comment:
Need new form since this form is for CNCF

Reply:
Jory Burson - need your help

Reply:
As I stated earlier, I don't think we should use Google forms for this. A PR is all we need.